### ISSUE TITLE
add option to disable scrapping enchantment 

### ISSUE THREAD

**Issue by majijn** [gigaherz/Survivalist#72]
I set every single option under "scraping" as false and i am still getting the enchantment , it stops me from seeing if I'm going to get other enchants (for example fortune, which is the one I am currently unable to get)
it may even be messing with the vanilla multiple enchantments algorithm , because I should have already gotten a fortune book from my AFK fish farm (ive left it on overnight for three nights, I dont know how I haven't gotten any level of fortune, given how ive gotten several silk touch weapons and tools

I wouldnt mind being able to have the option to have it be a treasure enchantment rather than a normal one, as it's mostly the fact that it's so ubiquitous when enchanting things through the table rather than an anvil that's proving so troublesome to me

**Comment by gigaherz** [comment]
Oops, when scraping is disabled, it would still have the enchantment itself, just it would do nothing.

I have now changed it so that the enchantment keeps existing (in case existing items use it), but if it's disabled, it can't be applied to anything (meaning it won't show on the enchantment table, and the books won't work on the anvil... except in creative mode).

I have also added a "IsTreasureEnchantment" option, since I thought it's a good idea.
